src/Discord.Net.SlashCommands/RunMode.cs

```diff
@@ -0,0 +1,9 @@
+namespace Discord.SlashCommands
+{
+    public enum RunMode
+    {
+        Sync,
+        Async,
+        Default
```

Comment:
Default seems redundant?

Reply:
It is required for changing the runmode behaviour of a command from its attribute. This pattern is directly ported from the `Commands` project.